doc/changelog/07-commands-and-options/xxxxx-show-script.rst

```diff
@@ -0,0 +1,2 @@
+- Remove :n:`Show Script` command (deprecated since 8.10)
```

Comment:
You don't actually need the `:n:` since there is no notation there, so:

```suggestion
- Remove ``Show Script`` command (deprecated since 8.10)
```

would also work.